pkgs/os-specific/windows/sdks/default.nix

```diff
@@ -0,0 +1,264 @@
+{ stdenvNoCC, fetchurl, lib, config, msitools, unzip, static ? false }:
+
+#if !(config.microsoftVisualStudioLicenseAccepted or false)
+if false
```

Comment:
?

Reply:
Temporary while I get things to work. We should add another line of defense beyond things being broken and not part of the default jobs so @GrahamcOfBorg doesn't do anything with those builds.